### PR TITLE
Feature/572 group new model ui email

### PR DESCRIPTION
**Pull request by quiteqiang**
Added "reply-to" feature

The recipient is able to reply to the email to a different person than the sender.

### CODE REVIEW

File: api/src/emails/api.py [hunk 1]
```diff
@@ -22,7 +22,9 @@ def send_email():
         return jsonify(err.messages), 422
 
     msg = Message(valid_email_request['subject'], recipients=valid_email_request['recipients'],
-                                                   sender=valid_email_request['managerEmail'])
+                                                   sender=valid_email_request['managerEmail'],
+#                                                    reply_to='qiang_wang@taylor.edu')
```
Comment (nurkkala) [inline api/src/emails/api.py]: Should this comment remain in the code?

File: ui/src/components/EmailForm.vue [hunk 2]
```diff
@@ -7,9 +7,6 @@
         <h3 class="headline mb-0">{{ $t("groups.members.email.compose") }}</h3>
       </div>
     </v-card-title>
-    <!-- TODO: Can we use a loop to render the following 3 components?
-      Currently the issue is that v-model won't synchronize references
-      that are returned by computed properties <2020-06-17, David Deng> -->
```
Comment (nurkkala) [inline ui/src/components/EmailForm.vue:12]: Was this concern addressed?

File: ui/src/components/EmailForm.vue [hunk 3]
```diff
@@ -78,27 +75,125 @@
...
+                  </v-card-text>
+                  <v-btn v-on:click="hideManagerPanel" color="light-blue" flat>{{
+                    $t("actions.cancel")
+                    }}</v-btn>
+                  <v-btn v-on:click="setReplyTo" color="primary" flat>{{
+                    $t("actions.confirm")
+                    }}</v-btn>
+                </v-card>
+              </v-expand-transition>
+<!--            <v-expand-transition>-->
```
Comment (nurkkala) [inline ui/src/components/EmailForm.vue]: Do we want to retain this commented-out markup?

File: ui/src/components/EmailForm.vue [hunk 4]
```diff
@@ -78,27 +75,125 @@
...
+          </v-card-actions>
+        </v-card>
+      </v-expand-transition>
+    </v-flex>
     <v-card-actions>
       <v-btn v-on:click="cancel" color="secondary" flat>{{
         $t("actions.cancel")
-      }}</v-btn>
+      }}123</v-btn>
+      <v-spacer>
```
Comment (nurkkala) [inline ui/src/components/EmailForm.vue:158]: Empty elements can be changed to use "self closing": `<v-spacer />`

File: ui/src/components/EmailForm.vue [hunk 5]
```diff
@@ -174,9 +315,33 @@ export default {
         cc: [],
         bcc: [],
         managerName: "", // TODO: use this field in the composed email later
```
Comment (nurkkala) [inline ui/src/components/EmailForm.vue]: Are these TODOs resolved or should they stay in the code? If the latter, create an issue so we get back to them.

File: ui/src/components/EmailForm.vue [hunk 6]
```diff
@@ -174,9 +315,33 @@ export default {
...
-        managerEmail: "sender@xx.com" // TODO: ask the user to choose the sender email
-      }
+        managerEmail: "manager@xx.com", // TODO: ask the user to choose the sender email
+        reply_to:""
+      },
+      expand:false,
+      entityTypePanel: {
```
Comment (nurkkala) [inline ui/src/components/EmailForm.vue:307]: Making these objects suggests that there is more state being managed per panel. If so, fine, but if not, consider making these attributes simple Boolean values. For example `showEntityTypePanel: false`, etc. (I'm guessing you've followed a pattern seen elsewhere in the code.)

**Review by nurkkala — COMMENTED** [review]
Looks very good. Please address the minor questions/issues I've posed.

File: ui/src/components/EmailForm.vue (hunk 2 above)
Comment (quiteqiang) [reply ui/src/components/EmailForm.vue:12]: Yes, we figured out that this issue could be solved by coding the 3 components separately.

File: ui/src/components/EmailForm.vue (hunk 3 above)
Comment (quiteqiang) [reply ui/src/components/EmailForm.vue]: They should be removed, that was a mistake I made. Sorry about this.

File: ui/src/components/EmailForm.vue (hunk 5 above)
Comment (quiteqiang) [reply ui/src/components/EmailForm.vue]: This piece of code might be needed for the Issue-550. [https://github.com/corpus-christi/corpus-christi/issues/550](url), the issue is updated now

File: ui/src/components/EmailForm.vue (hunk 6 above)
Comment (quiteqiang) [reply ui/src/components/EmailForm.vue:307]: Ok, I see. I will change that coding pattern in the future.

File: api/src/emails/api.py (hunk 1 above)
Comment (quiteqiang) [reply api/src/emails/api.py]: This should be removed, which is for testing.

**Review by nurkkala — APPROVED** [review]

**Pull request merged**